sdk/Cargo.toml

```
@@ -53,6 +53,7 @@ rocksdb = { version = "0.20.1", default-features = false, features = [ "lz4" ],
 rumqttc = { version = "0.20.0", default-features = false, features = [ "websocket" ], optional = true }
 serde-big-array = { version = "0.5.1", default-features = false, optional = true }
 serde_repr = { version = "0.1.12", default-features = false, optional = true }
+time = { version = "0.3.20", default-features = false, features = [ "serde", "macros" ], optional = true }
 thiserror = { version = "1.0.40", default-features = false, optional = true }
```

```suggestion
thiserror = { version = "1.0.40", default-features = false, optional = true }
time = { version = "0.3.20", default-features = false, features = [ "serde", "macros" ], optional = true }
```
Please don't hate me